src/com/library/Entities/Member.java

```diff
@@ -0,0 +1,82 @@
+package com.library.Entities;
+
+public class Member {
+    private int id;
+    private String firstName;
+    private String lastName;
+    private String phone;
+    private String email;
+    private Address address;
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public void setFirstName(String firstName) {
+        this.firstName = firstName;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
+
+    public void setLastName(String lastName) {
+        this.lastName = lastName;
+    }
+
+    public String getPhone() {
+        return phone;
+    }
+
+    public void setPhone(String phone) {
+        this.phone = phone;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public void setEmail(String email) {
+        this.email = email;
+    }
+
+    public Address getAddress() {
+        return address;
+    }
+
+    public void setAddress(Address address) {
+        this.address = address;
+    }
+
+    public Member() {
```

Comment:
[Fix] ordinea in clasa este: campuri, constructori, getteri + setteri, nu le intercala, conventia este in acest fel ca sa stie lumea unde gaseste informatia corecta, si nu sa faca scroll prin clasa sa gaseasca ce ii intereseaza.

la fel si pentru celelalte clase